cmake/Fortran.cmake

```diff
@@ -102,12 +102,14 @@ elseif(UNIX)
   if(CMAKE_Fortran_COMPILER MATCHES "ifort")
     target_compile_options(fortran_project_options INTERFACE -fpp)
     target_link_options(fortran_project_options INTERFACE -static-intel)
+    target_compile_options(fortran_project_options INTERFACE $<$<CONFIG:Debug>:-traceback>)
   else()
     if(NOT "Ninja" STREQUAL ${CMAKE_GENERATOR})
       target_compile_options(fortran_project_options INTERFACE -cpp)
     endif()
     set(FORTRAN_STATIC_EXE TRUE)
     target_link_options(fortran_project_options INTERFACE -static)
+    target_compile_options(fortran_project_options INTERFACE $<$<CONFIG:Debug>:-fbacktrace -ffpe-trap=zero,overflow,underflow>)
```

Comment:
Pass some debug options to gfortran / ifort when building debug. I havern't found much differences but it makes sense